Objects/memoryobject.c

```diff
@@ -3303,10 +3305,53 @@ typedef struct {
     const char *it_fmt;
 } memoryiterobject;
 
+#define _PyMemoryViewIter_CAST(PTR) ((memoryiterobject *)(PTR))
+
```

Comment:
Not needed anymore
```suggestion
```

Reply:
Typically, they go in the header file then. It's not particularly useful if it's only used from C code.

Reply:
Well.. that's also what we did. Only in the C file. Not wanting to expose them yet.